vtex-apps/menu
# Add list context support

#### What is the purpose of this pull request?
As title says

#### How should this be manually tested?
1. Go to the [workspace](https://menulisttest--alssports.myvtex.com/admin/cms/site-editor) 
2. Go to `Header` > `Menu` > `Men's` > `Submenu` > click on the fisrt `Menu`
3. Add items and see the change

#### Types of changes
